eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -36,6 +38,11 @@
  */
 final class HashOfTag {
 
+    /**
+     * Static map with all hash requests.
+     */
+    private static final Map<String, String> REQUESTED_HASHES = new HashMap<>();
```

Comment:
@EugeneDar maybe we should make this implementation thread safe? It's important for Maven.

Reply:
@yegor256 yes, I'll fix it.

Reply:
@yegor256 I'll use `ConcurrentHashMap`.

Reply:
@EugeneDar good idea